app/assets/sass/application.scss

```diff
@@ -62,6 +62,10 @@ table.no-pad-right td {
     margin-top: 40px;
 }
 
+.table-short-column {
+    width: 200px !important;
```

Comment:
Ideally best to try and avoid using `!important` but can sometimes be least worst option when trying to work with a codebase like this...

Reply:
Thanks @tpmcgowan. I think this was there whilst i was trying to override something else. Turns out its not needed so have removed! 😄